front/app/modules/commercial/verification/citizen/components/VerificationMethodButton.tsx

```diff
@@ -1,35 +1,26 @@
-import { Button, ButtonProps } from '@citizenlab/cl2-component-library';
-import styled from 'styled-components';
+import { Box, Button, ButtonProps } from '@citizenlab/cl2-component-library';
 import React, { ReactNode } from 'react';
-import useLocale from 'hooks/useLocale';
-import { isNilOrError } from 'utils/helperUtils';
-
-const MethodButton = styled(Button)<{ last: boolean }>`
-  margin-bottom: ${({ last }) => (last ? '0px' : '15px')};
-`;
 
 interface Props extends Partial<ButtonProps> {
   children: ReactNode;
   last: boolean;
 }
 
 const VerificationMethodButton = (props: Props) => {
-  const locale = useLocale();
-  if (isNilOrError(locale)) return null;
-
   return (
-    <MethodButton
-      icon="shieldVerified"
-      iconSize="22px"
-      buttonStyle="white"
-      fullWidth={true}
-      justify="left"
-      whiteSpace="wrap"
-      locale={locale}
-      {...props}
-    >
-      {props.children}
-    </MethodButton>
+    <Box mb={props.last ? '0px' : '15px'}>
+      <Button
+        icon="shield-check"
+        iconSize="22px"
```

Comment:
Normally, we shouldn't need custom icon sizes anymore.

Reply:
Just copied and pasted what was there, but ok 👍